src/Entity/Article.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ArticleRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use App\Entity\Image;
+
+/**
+ * @ORM\Entity(repositoryClass=ArticleRepository::class)
+ */
+class Article
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private int $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $titre;
+
+    /**
+     * @ORM\Column(type="date", nullable=true)
+     */
+    private \DateTimeInterface $date;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private string $description;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $section;
+
+    /**
+     * @ORM\ManyToMany(targetEntity=Image::class, inversedBy="articles")
+     */
+    private collection $image;
```

Comment:
It probably should be Collection (with capital C because it's a class)

Reply:
Ok, check.